src/framework/preferences/js/AuxBuilder.js

```diff
@@ -349,10 +341,17 @@ var fluid_2_0 = fluid_2_0 || {};
         return auxSchema;
     };
 
+    // Go over the aux schema to replace terms with their given values.
     fluid.prefs.expandSchema = function (schemaToExpand, indexes, topCommonOptions, elementCommonOptions, mappedDefaults) {
         var auxSchema = fluid.prefs.expandSchemaImpl(schemaToExpand);
         auxSchema.namespace = auxSchema.namespace || "fluid.prefs.created_" + fluid.allocateGuid();
 
+        var terms = fluid.get(auxSchema, "terms");
+        if (terms) {
+            delete auxSchema.terms;
+            fluid.set(auxSchema, ["terms", "terms"], terms);
```

Comment:
"terms" is a component constructed and used internally by the builder. 

The basic definition for it - https://github.com/cindyli/infusion/blob/FLUID-5598/src/framework/preferences/js/AuxBuilder.js#L420-L422. The "terms" from the aux schema is passed in as an option.

 Once constructed, this component is used as a grade for constructing prefsEditorLoader (https://github.com/cindyli/infusion/blob/FLUID-5598/src/framework/preferences/js/Builder.js#L122) where its terms option is passed down to loaders.